cloud-core/src/main/java/cloud/commandframework/internal/CommandRegistrationHandler.java

```diff
@@ -54,6 +55,16 @@ public interface CommandRegistrationHandler {
      */
     boolean registerCommand(@NonNull Command<?> command);
 
+    /**
+     * Requests that the given {@code rootCommand} should be unregistered.
+     *
+     * @param rootCommand The command to delete
+     * @since 1.7.0
+     */
+    default void unregisterRootCommand(final @NonNull StaticArgument<?> rootCommand) {
```

Comment:
Should this default to a no-op instead? the feature is already guarded by a capability for managers.